src/h/config.h

```diff
@@ -548,6 +548,8 @@
 #endif					/* IntBits */
 
 #if (WordBits == 64)
+   #define LINTFRMT "l"
```

Comment:
My first thought on seeing this line was "What's lint got to do with it?".  It was only later when looking at rmisc.r that I realised it meant "Long Integer".  C99 introduced the inttypes.h header with predefined printf format specifiers (PRId64 etc.)  Is it worth considering using that?

Reply:
Given your comment below, no action is needed here then.